File: sanidad-backend/src/main/java/com/farmacia/sanidadbackend/dto/RecetaResponse.java
package com.farmacia.sanidadbackend.dto;

import lombok.Data;
import java.time.LocalDateTime;

@Data
public class RecetaResponse {
    private Long id;
    private String imagenUrl;
    private LocalDateTime fechaSubida;
    private String estado;
    private Long farmaceuticoId;
    private String farmaceuticoUsername;
    private Long ventaId;
}

File: SanidadMovil/app/src/main/java/com/sanidad/movil/data/remote/dto/RecetaResponse.kt
package com.sanidad.movil.data.remote.dto

data class RecetaResponse(
    val id: Long,
    val imagenUrl: String?,
    val fechaSubida: String?,
    val estado: String,
    val farmaceuticoId: Long,
    val farmaceuticoUsername: String?,
    val ventaId: Long?,
    val codigoMinsa: String?
)